examples/call-http-shorthand-endpoint.yaml

```
@@ -0,0 +1,11 @@
+document:
+  dsl: 1.0.0-alpha1
+  namespace: examples
+  name: call-http-shorthand-endpoint
+  version: 1.0.0-alpha1
+do:
+  getPetById:
+    call: http
+    with:
+      method: get
+      endpoint: https://petstore.swagger.io/v2/pet/1
```

```suggestion
      endpoint: https://petstore.swagger.io/v2/pet/{petId}
```
If you change to a template, the CI will fail since the format below is `uri`.